CODE_OF_CONDUCT.md

```diff
@@ -1,55 +1,36 @@
-# Code of Conduct
-To be a truly great community, SwiftNIO needs to welcome developers from all walks of life,
-with different backgrounds, and with a wide range of experience. A diverse and friendly
-community will have more great ideas, more unique perspectives, and produce more great
-code. We will work diligently to make the SwiftNIO community welcoming to everyone.
-
-To give clarity of what is expected of our members, SwiftNIO has adopted the code of conduct
-defined by [contributor-covenant.org](https://www.contributor-covenant.org). This document is used across many open source
-communities, and we think it articulates our values well. The full text is copied below:
-
-### Contributor Code of Conduct v1.3
-As contributors and maintainers of this project, and in the interest of fostering an open and
-welcoming community, we pledge to respect all people who contribute through reporting
-issues, posting feature requests, updating documentation, submitting pull requests or patches,
-and other activities.
-
-We are committed to making participation in this project a harassment-free experience for
-everyone, regardless of level of experience, gender, gender identity and expression, sexual
-orientation, disability, personal appearance, body size, race, ethnicity, age, religion, or
-nationality.
+To be a truly great community, the SwiftNIO project needs to welcome developers from all walks of life, with different backgrounds, and with a wide range of experience. A diverse and friendly community will have more great ideas, more unique perspectives, and produce more great code.  We will work diligently to make the SwiftNIO community welcoming to everyone.
```

Comment:
We're missing `# Code of Conduct` now -- is that intentional?

Reply:
@glbrntt no! thank you, updating all.

Reply:
great catch, should be fixed. thanks